user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -68,9 +68,9 @@ For a self-hosted DataMiner System, follow the steps below to set up STaaS.
 
 DataMiner STaaS relies on Azure Storage, which stores multiple copies of your data to make sure it is always available even in case outages or disasters occur. Different storage redundancy setups are possible. STaaS supports zone-redundant storage and geo-redundant storage. When you contact Skyline to register your system to use STaaS, you can include your preferences as to the region(s) where your data should be stored and the type of storage redundancy that should be used.
 
-- **Zone-redundant storage (ZRS)** copies your data synchronously across three Azure availability zones in one region. Each availability zone is a separate physical location with independent power, cooling, and networking.
+- **Zone-redundant storage (ZRS)** copies your data synchronously across three Azure availability zones in one region. Each availability zone is a separate physical location with independent power, cooling, and networking. By **default**, DataMiner STaaS uses ZRS.
 
-- **Geo-redundant storage (GRS)** copies your data synchronously three times within a single physical location in the primary region and then also copies your data asynchronously to a single physical location in the secondary region. Only specific regions can be combined in such a setup, e.g. if the primary region is Switzerland North, the secondary region can only be Switzerland West. For an overview of the supported regions, see [Azure paired regions](https://learn.microsoft.com/en-us/azure/reliability/cross-region-replication-azure#azure-paired-regions).
+- **Geo-redundant storage (GRS)** copies your data synchronously three times within a single physical location in the primary region and then also copies your data asynchronously to a single physical location in the secondary region. Only specific regions can be combined in such a setup, e.g. if the primary region is Switzerland North, the secondary region can only be Switzerland West. For an overview of the supported regions, see [Azure paired regions](https://learn.microsoft.com/en-us/azure/reliability/cross-region-replication-azure#azure-paired-regions). If you wish to use DataMiner STaaS with GRS, contact <daas@dataminer.services>.
```

Comment:
@BertVandenberghe, I've used the daas@dataminer.services email address based on your feedback; however, everywhere else on this page we refer to staas@dataminer.services. Is this OK as it is, or should it be the STaaS address here as well?

Reply:
Sorry, you should then indeed best use staas@dataminer.services here, my bad. Both addresses exist and they will both help you out, but it is more correct to use the staas one here because we are talking about a STaaS config which could be used even without DaaS.